core/include/spi.h

```diff
@@ -56,5 +56,7 @@ struct spi_ops {
 		size_t *num_rxpkts);
 };
 
+void spi_init(void);
```

Comment:
How SPI is initialized seems to be platform specific for the moment so please keep that under `core/arch/arm/plat-hikey`